### ISSUE TITLE
Push signed data on-chain

### ISSUE THREAD

**Issue by Siegrift** [api3dao/airseeker-v2#9]
After we query the signed data, we need to publish it on-chain using the corresponding sponsor wallet derived from the dapi name.

To be done after #18

**Comment by Siegrift** [comment]
Actually, this one is a duplicate of https://github.com/api3dao/airseeker-v2/issues/36
